docs/source/executing-operations/demand-control.mdx

```diff
@@ -7,523 +7,205 @@ minVersion: 1.48.0
-
-```yaml title="router.yaml"
-telemetry:
-  instrumentation:
-    instruments:
-      supergraph:
-        # custom instrument
-        cost.rejected.operations:
-          type: histogram
-          value:
-            # Estimated cost is used to populate the histogram
-            cost: estimated
-          description: "Estimated cost per rejected operation."
-          unit: delta
-          condition:
-            eq:
-              # Only show rejected operations.
-              - cost: result
-              - "COST_ESTIMATED_TOO_EXPENSIVE"
-          attributes:
-            graphql.operation.name: true                             # Graphql operation name is added as an attribute
-
-```
-
-This custom instrument may not be suitable when you have many operation names, such as a public internet-facing API. You can add conditions to reduce the number of returned operations. For example, use a condition that outputs results only when the cost delta is greater than a threshold:
-
-```yaml title="router.yaml"
-telemetry:
-  instrumentation:
-    instruments:
-      supergraph:
-        # custom instrument
-        cost.rejected.operations:
-          type: histogram
-          value:
-            # Estimated cost is used to populate the histogram
-            cost: estimated
-          description: "Estimated cost per rejected operation."
-          unit: delta
-          condition:
-            all:
-              - eq: # Only show rejected operations
-                 - cost: result
-                 - "COST_ESTIMATED_TOO_EXPENSIVE"
-#highlight-start
-              - gt: # Only show cost delta > 100
-                 - cost: delta
-                 - 100
-#highlight-end
-```
+</ExpansionPanel>
 
-4. You should now be able to configure your APM to see which operations are too costly. Visualizing the histogram can be useful, such as with top-N or heatmap tools.
+### Customizing cost
 
-For example, the following table has the estimated cost of operations:
+Since version 1.53.0, the router supports customizing the cost calculation with the `@cost` directive. The `@cost` directive has a single argument, `weight`, which overrides the default weights from the table above.
 
-| Operation name       | Estimated cost |
-|----------------------|----------------|
-| `ExtractAll`         | 9020           |
-| `GetAllProducts`     | 1435           |
-| `GetLatestProducts`  | 120            |
-| `GetRecentlyUpdated` | 99             |
-| `FindProductByName`  | 87             |
+<Note>
 
-The `ExtractAll` operation has a very large estimated cost, so it's a good candidate to be rejected.
+The Apollo Federation [@cost directive](/federation/federated-schemas/federated-directives/#cost) differs the IBM specification in that the `weight` argument is of type `Int!` instead of `String!`.
```

Comment:
```suggestion
The Apollo Federation [`@cost` directive](/federation/federated-schemas/federated-directives/#cost) differs from the IBM specification in that the `weight` argument is of type `Int!` instead of `String!`.
```